.github/workflows/release.yaml

```
@@ -25,6 +25,8 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
+      - name: free disk space
+        uses: jlumbroso/free-disk-space@f68fdb76e2ea636224182cfb7377ff9a1708f9b8 # v1.3.0
```

To not break self-hosted runners occasionally, I would add a condition like this (a bit ugly, but I didn't find a way to access `runs-on:` from `if:`):

```suggestion
    strategy:
      matrix:
        os: [ ubuntu-latest ]
    runs-on: ${{ matrix.os }}
    steps:
      - name: free disk space
        if: matrix.os == 'ubuntu-latest'
        uses: jlumbroso/free-disk-space@f68fdb76e2ea636224182cfb7377ff9a1708f9b8 # v1.3.0
```